doc/installation.rst

```diff
@@ -2,21 +2,31 @@
 Installation
 ############
 
-If you intend to use EOS without any modifications, then you should follow the steps outlined in `Installing with a Package Manager`_
-or `Installing with the Python Package Index (PyPI)`_.
-If you intend to modify EOS for your personal purpose, then you must follow the steps outlined in `Installing from Source`_.
+We distribute EOS as packaged software for end-users and also as source code for developers.
 
 
-*********************************
-Installing with a Package Manager
-*********************************
+*****
+Linux
+*****
+
+Installation with `pip3`
+========================
+
+This is the recommended installation method using the package installer `pip3`:
```

Comment:
installer -> manager? That's why they are usually called!

Reply:
It's a specific (not generic) name, but we can set it in quotation marks.